doc/topics/development/releasenotes.rst

```diff
@@ -0,0 +1,71 @@
+.. _releasenotes:
+
+=============
+Release Notes
+=============
+
+You can edit the release notes to highlight a new feature being added
+to a given release. The release notes are templatized with Jinja and
+are generated at release time.
+
+
+.. _edit-release-note:
+
+How do I edit the release notes
+-------------------------------
+
+To edit the release notes you need to look in doc/topics/releases/templates
+for your given release and edit the template. Do not edit the release note
+files in doc/topics/releases/, as this will be written over with the content
+in the template file. For example, if you want to add content to the 3006.0
+release notes you would edit the doc/topics/releases/templates/3006.0.md.template
+file. Do not edit the changelog portion of the template file, since that is
+auto generated with the content generated for the changelog for each release.
```

Comment:
So, to clarify, this isn't to edit the actual release notes, but to edit the introductory section before the automated changelog output?

Reply:
Correct, the release notes will be generated at release time